lessons/04-client-jquery-ajax/HOMEWORK.md

```diff
@@ -0,0 +1,46 @@
+### Preclass Reading and Exercises **for this class (class 05), TODO- put in class 04 homework**
+- Read the [Class 5 README](https://github.com/olinjs/olinjs/blob/master/lessons/04-client-jquery-ajax/README.md). There's parts to follow along, so please do so. Push your app with the 3 forms to your fork of the olinjs repo and send us a link to the folder on github. It is pretty long, so start early. If you really need to, at least get to where you have 2 forms working, and work through the rest of the README in class if you need to.  
```

Comment:
Is the app you're referring to part of the Class 5 readme, or the below assignment. This is a bit vague and confusing to me.